learn-regular-expressions-by-building-a-spam-filter/index.js

```diff
@@ -0,0 +1,40 @@
+const messageInput = document.getElementById("message-input");
+const result = document.getElementById("result");
+const checkMessageButton = document.getElementById("check-message-btn");
+
+/**
+ * I read through the previous discussions for this project
+ * and we can start with matches for literal phrases and then have the
+ * campers create more complex regex which seemed to be the original intent for this project.
+ *
+ *
+ *
+ */
+
+const blacklistRegexps = [
```

Comment:
Given the discourse around `whitelist`/`blacklist`, do we want to change this to `denyList`?

Reply:
Thanks for catching this @naomi-lgbt.

I agree. `denyList`, `blockList`, or something like `denyListRegexps` would be a better variable name.

Reply:
I like denyListRegexps 👍